src/main/java/com/vmware/vipclient/i18n/messages/service/StringService.java

```diff
@@ -36,19 +36,21 @@ public String getString(MessagesDTO dto) {
     	Iterator<Locale> fallbackLocalesIter = LocaleUtility.getFallbackLocales().iterator();
     	MessageCacheItem cacheItem = new ComponentService(dto).getMessages();
     	
-    	// While failed to get MessageCacheItem, use MessageCacheItem of the next fallback locale. 
-    	while (cacheItem.getCachedData().isEmpty() && fallbackLocalesIter.hasNext()) {
-    		Locale fallback = fallbackLocalesIter.next();
-			MessagesDTO fallbackLocaleDTO = new MessagesDTO(dto.getComponent(), 
-					key, dto.getSource(), fallback.toLanguageTag(), null);
-			cacheItem = new ComponentService(fallbackLocaleDTO).getMessages();
-			
-			// Cache a reference to the MessageCacheItem of the fallback locale 
-			if (!cacheItem.getCachedData().isEmpty()) {
-				CacheService cacheService = new CacheService(dto);
-				cacheService.addCacheOfComponent(cacheItem);
+    	if (VIPCfg.getInstance().isProdMode()) {
```

Comment:
If statement added for use case #1: Requested locale is not supported (bundle not found onlline or offline)